manager/pom.xml

```diff
@@ -145,7 +145,6 @@
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <scope>provided</scope>
         </dependency>
```

Comment:
hi, can not depend the mysql jar when package

Reply:
Changed it. Accidentally uploaded the local code